release notes/v0.48.0.md

```diff
@@ -0,0 +1,256 @@
+- [browser#1078](https://github.com/grafana/xk6-browser/pull/1078) fixes request interception deadlock to improve stability.
+- [browser#1101](https://github.com/grafana/xk6-browser/pull/1101) fixes `page.$` so that it returns `null` when no matches with given selector are found.
+- [#3397](https://github.com/grafana/k6/pull/3397), [#3427](https://github.com/grafana/k6/pull/3427), [#3417](https://github.com/grafana/k6/pull/3417) update `goja` dependency. Fixes a possible panic and proper handling circular types at `JSON.stringify`. Fixes an issue about dumping the correct stack trace when an error is re-thrown.
+
+## Maintenance and internal improvements
+
+- [#3378](https://github.com/grafana/k6/pull/3378) fixes usage of `gh` in GitHub actions creating the OSS release.
+- [#3386](https://github.com/grafana/k6/pull/3386), [#3387](https://github.com/grafana/k6/pull/3387), [#3388](https://github.com/grafana/k6/pull/3388), [browser#1047](https://github.com/grafana/xk6-browser/pull/1047) update dependencies.
+- [#3393](https://github.com/grafana/k6/pull/3393), [#3399](https://github.com/grafana/k6/pull/3399) fix lint issues in the `js` package.
+- [#3381](https://github.com/grafana/k6/pull/3381) disables temporarily ARM tests on GitHub Actions.
+- [#3401](https://github.com/grafana/k6/pull/3401), [#3469](https://github.com/grafana/k6/pull/3469) refactors a Makefile, removes `make ci-like-lint` in favor of `make lint`. Updates a golangci-lint version to v1.55.2.
+- [#3410](https://github.com/grafana/k6/pull/3410) fixes the `tests` reference in the all rule of the Makefile. Thanks, @flyck!
+- [#3402](https://github.com/grafana/k6/pull/3402) adds a test-case for the `k6 cloud`.
+- [#3421](https://github.com/grafana/k6/pull/3421) updates dependencies for xk6 integration tests.
+- [browser#1075](https://github.com/grafana/xk6-browser/pull/1075), [browser#1076](https://github.com/grafana/xk6-browser/pull/1076) refactors `clearPermissions` and `grantPermissions`.
+- [browser#1043](https://github.com/grafana/xk6-browser/pull/1043) refines tests.
+- [browser#1069](https://github.com/grafana/xk6-browser/pull/1069), [browser#1090](https://github.com/grafana/xk6-browser/pull/1090) refactor internal.
+- [browser#1102](https://github.com/grafana/xk6-browser/pull/1102) uses `force` and `noWaitAfter` in `frame.newAction`.
+- [#3443](https://github.com/grafana/k6/pull/3443) mentions that k6-core team aims to support the last two major golang versions for building a k6 binary.
+- [#3437](https://github.com/grafana/k6/pull/3437) switches k6 cloud traces to a new hostname.
+- [#3429](https://github.com/grafana/k6/pull/3429) increases timeout expectations for the `TestSetupTimeout` test.
+- [#3446](https://github.com/grafana/k6/pull/3446) moves log tokenizer to `lib/strvals` package.
+
+## _Optional_ Roadmap
+
+_Discussion of future plans_
+
```

Comment:
What is the plan here? Are we waiting for the upcoming planning session?

Reply:
Yep, you've got it right :wink:

Reply:
@oleiade pitched to me the ideas for `xk6-dashboard` maybe we can mention it here, it should also give some visibility to the project. @olegbespalov @oleiade  WDYT?

Reply:
I'm adding it, sorry for the delay folks 🙇🏻

Reply:
I've added commit 92dbc0b336106a1a5d101106b07869c104f50166 with a dedicated small subsection to the Roadmap 👍🏻